great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -297,11 +294,16 @@ def _init_factories(self) -> None:
         )
 
     def _init_analytics(self) -> None:
+        enabled_from_config = self.config.analytics
+        if enabled_from_config is None:
```

Comment:
Same thoughts as above - I don't understand how `enabled_from_config` ties to `analytics`, and we should avoid `None` if we can just ensure it's instantiated with a default `True`